sale_report_hide_price/tests/test_sale.py

```diff
@@ -12,8 +12,8 @@ def test_saleorder_action(self):
         self.sale_order = self.env['sale.order']
         self.partner = self.env.ref('base.res_partner_2')
         self.products = {
-            'serv_order': self.env.ref('product.service_order_01'),
-            'serv_deli': self.env.ref('product.service_delivery'),
+            'serv_order': self.env.ref('product.product_delivery_01'),
+            'serv_deli': self.env.ref('product.product_order_01'),
```

Comment:
you made sure this XML ID is rigt?

Reply:
@JayVora-SerpentCS 
yes sir. it's right. i was checked in base. previously XML ID gave error in test cases.